light-client/relayer/src/sender.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018-2023 The MobileCoin Foundation
+
+use mc_blockchain_types::{Block, BlockContents, BlockMetadata};
+use mc_common::logger::{log, Logger};
+use mc_transaction_core::tx::TxOut;
+
+// A sync which can send interesting blocks and metadata found by the relayer to
+// its intended destination.
+pub trait Sender {
+    fn send(
+        &mut self,
+        outputs: Vec<TxOut>,
+        block: &Block,
+        block_contents: &BlockContents,
+        block_metadata: Vec<BlockMetadata>,
+    );
+}
+
+/// A dummy sender which just logs anything that is sent.
+pub struct DummySender {
```

Comment:
:upside_down_face: nit It may be my bad vocabulary but I think of "dummy" as not doing anything, while this appears to be logging

Perhaps:
```suggestion
/// A sender which just logs anything that is sent.
pub struct LoggingSender {
```

Reply:
I guess my point of view is that logging is functionally nothing, and DummySender more obviously is not the right thing to use in prod, while LoggingSender might be a wrapper or something that should be used in prod. 

Happy to change the name if this does not change your mind.

Reply:
I think it's fine as is.